server/static/registerDescription.json

```diff
@@ -0,0 +1,12 @@
+[   
```

Comment:
Instead the format could be:

```
{ "heading': "whatever"
, "content": [
    { "heading": "whatever smaller, no number included"
    , "content": ["paragraph", "second paragraph"]
    },
    ...
  ]
}
```

Should be easier to parse in to the previous format.